src/components/WebcamCapture/reduxStore.js

```diff
@@ -0,0 +1,36 @@
+import {createStore, applyMiddleware, combineReducers, compose} from 'redux';
```

Comment:
why has this redux store file been added here? Surely it is somewhere else?

Reply:
this is a mistake, accidentally copied the file from utils

Reply:
The renaming of the Webcam file is accidental, fixing it now